learn/app-development/deployment/deployment-to-digital-ocean.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Configuring Digital Ocean"
+id: ""
+sidebar_label: "Digital Ocean"
+---
+---
+
+DigitalOcean Kubernetes (DOKS) is a DigitalOcean managed Kubernetes service that lets you deploy Kubernetes clusters without the complexities of handling the container infrastructures. 
+For the other providers like AWS, Azure, and GoogleCloud, only VM based deployments are supported and in this case, the container-related infrastructures need to be managed and monitored continuously manually.
+
+### What is managing container infrastructure, and why do users prefer Kubernetes deployments?
+In modern architecture, there are multiple services that are delivered over the network via APIs (for ex: our WaveMaker platform requires and runs multiple services at a time). All these services are hosted on multiple servers and configurations in various locations all coordinating their actions via network communication. This kind of distributed architecture is useful for scaling up the services. 
+Now let's say. at any given point of time, service A is trying to contact service B. However Service B is down due to some infrastructure-related issues.
+
+## VM deployment
+The communication between Service A and B will fail until a dev-ops member manually goes and fixes the cause for the Service B's container failure.  
+The entries of the services and their endpoints is maintained in consul and need to be updated manually.
+
+## Kubernetes deployment
+
+Kubernetes automatically keeps performing a health check on all its containers and in case of any failure like this, it relaunches another container with the same configuration to prevent any service communication failures.
+Also, Kubernetes maintains the details of the services and their endpoints in a registry and each time a container is relaunched and re-allocated, the endpoint for the service is automatically updated. 
+Autoscaling is another feature that lets users scale their applications without a radical redesign of their existing systems.
+
+Hence, through this integration, we will be enabling the end-users to perform deployment operations on their DigitalOcean Kubernetes clusters.
+
+## Deployment flow
+
+Accept the DigitalOcean account token for the user. For more information, see [How to Create a Personal Access Token](https://www.digitalocean.com/docs/apis-clis/api/create-personal-access-token/). For testing, use the following token below:
+
+```
+836ae5a604051f8303efed9f46cf7a22ac369c562b7b786efa67829643ad74a5
```

Comment:
@krishnaraoch really we can share this token in the documentation?

Reply:
@PatnalaPriyanka is working on this documentation. It is not done, yet.

Reply:
We removed it.